### PR TITLE
Generate missing doc aliases

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: glib/src/gobject/flags.rs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 use crate::translate::*;
 
 bitflags::bitflags! {
+    #[doc(alias = "GParamFlags")]
     pub struct ParamFlags: u32 {
         const READABLE = 1;
```
Comment (bilelmoussaoui) [inline glib/src/gobject/flags.rs:8]: Would be nice if those are using the ffi values. Also they seem to lack a doc alias

Comment (GuillaumeGomez) [reply glib/src/gobject/flags.rs:8]: Ah indeed! One stone, two pigeons. :D (I need the first to get the second)

File: glib/src/gobject/flags.rs [hunk 2]
```diff
@@ -3,24 +3,30 @@
...
-        const USER_2 = 1024;
-        const USER_3 = 2048;
-        const USER_4 = 4096;
-        const USER_5 = 8192;
-        const USER_6 = 16384;
-        const USER_7 = 32768;
-        const USER_8 = 65536;
-        const EXPLICIT_NOTIFY = 1073741824;
-        const DEPRECATED = 2147483648;
+        #[doc(alias = "G_PARAM_READABLE as u32")]
```
Comment (bilelmoussaoui) [inline glib/src/gobject/flags.rs:8]: Hmmm

**Review by bilelmoussaoui — CHANGES_REQUESTED** [review]

File: glib/src/gobject/flags.rs (hunk 2 above)
Comment (GuillaumeGomez) [reply glib/src/gobject/flags.rs:8]: HAHA

Comment (bilelmoussaoui) [reply glib/src/gobject/flags.rs]: `alias.split()[0]` :p

Comment (GuillaumeGomez) [reply glib/src/gobject/flags.rs]: Something like that. ;)

**Review by bilelmoussaoui — APPROVED** [review]

File: glib/src/gobject/flags.rs [hunk 3]
```diff
@@ -3,24 +3,30 @@
...
-        const LAX_VALIDATION = 16;
-        const USER_0 = 128;
-        const USER_1 = 256;
-        const USER_2 = 1024;
-        const USER_3 = 2048;
-        const USER_4 = 4096;
-        const USER_5 = 8192;
-        const USER_6 = 16384;
-        const USER_7 = 32768;
-        const USER_8 = 65536;
```
Comment (sdroege) [inline glib/src/gobject/flags.rs:21]: Please add these back, thanks :)

File: glib/src/gobject/flags.rs [hunk 4]
```diff
@@ -3,24 +3,30 @@
...
+        #[doc(alias = "G_PARAM_READWRITE")]
+        const READWRITE = gobject_ffi::G_PARAM_READWRITE as u32;
+        #[doc(alias = "G_PARAM_CONSTRUCT")]
+        const CONSTRUCT = gobject_ffi::G_PARAM_CONSTRUCT as u32;
+        #[doc(alias = "G_PARAM_CONSTRUCT_ONLY")]
+        const CONSTRUCT_ONLY = gobject_ffi::G_PARAM_CONSTRUCT_ONLY as u32;
+        #[doc(alias = "G_PARAM_LAX_VALIDATION")]
+        const LAX_VALIDATION = gobject_ffi::G_PARAM_LAX_VALIDATION as u32;
+        #[doc(alias = "G_PARAM_STATIC_NAME")]
+        const STATIC_NAME = gobject_ffi::G_PARAM_STATIC_NAME as u32;
```
Comment (sdroege) [inline glib/src/gobject/flags.rs:21]: This makes no sense from Rust, same for the other `G_PARAM_STATIC_...` flags

Comment (GuillaumeGomez) [reply glib/src/gobject/flags.rs:21]: Ok§

File: glib/src/gobject/flags.rs [hunk 5]
```diff
@@ -3,13 +3,20 @@
...
+        const WRITABLE = gobject_ffi::G_PARAM_WRITABLE as u32;
+        #[doc(alias = "G_PARAM_READWRITE")]
+        const READWRITE = gobject_ffi::G_PARAM_READWRITE as u32;
+        #[doc(alias = "G_PARAM_CONSTRUCT")]
+        const CONSTRUCT = gobject_ffi::G_PARAM_CONSTRUCT as u32;
+        #[doc(alias = "G_PARAM_CONSTRUCT_ONLY")]
+        const CONSTRUCT_ONLY = gobject_ffi::G_PARAM_CONSTRUCT_ONLY as u32;
+        #[doc(alias = "G_PARAM_LAX_VALIDATION")]
+        const LAX_VALIDATION = gobject_ffi::G_PARAM_LAX_VALIDATION as u32;
         const USER_0 = 128;
```
Comment (sdroege) [inline glib/src/gobject/flags.rs:20]: Actually `USER_0` makes no sense. It starts at `1<<8`, which is `USER_1 = 256`

Comment (GuillaumeGomez) [reply glib/src/gobject/flags.rs]: Removed.

**Review by sdroege — APPROVED** [review]

**Pull request merged**